content/community/raising-issues/Types of Issue Labels.adoc

```diff
@@ -0,0 +1,57 @@
+---
+Title: Types of Issue Labels
+Weight: 20
+---
+
+Types of Issue Labels
+|===
+|Label name |[gray-background]#salesagility/suiteCRM# |Description
+
+|[gray-background]#bug#
+|https://github.com/salesagility/SuiteCRM/labels/bug[search]
+|Confirmed or very likely to be a bug.
+
+|[gray-background]#Low priority#
```

Comment:
I would also avoid the `#` formatting since this can be achieved with column styles.